web-app/src/components/BookingModal/index.js

```diff
@@ -54,31 +113,18 @@ const BookingModal = props => {
             toggleRejectionMessageInputView={toggleRejectionMessageInputView}
           />
         )}
-        <h1>
-          {isEventBeingUpdated ? 'Update Booking' : 'Request a Holiday'}
-        </h1>
-        <FormContainer>
-          <BookingModalForm
-            toggleRejectionResponseView={toggleRejectionResponseView}
-            updateTakenEvents={updateTakenEvents}
-            employeeId={employeeId}
-            booking={booking}
-            bookingDuration={bookingDuration}
-            createEvent={createEvent}
-            updateEvent={updateEvent}
-          />
-        </FormContainer>
+        {renderLegacyMesasge()}
+        {renderBookingModalForm()}
```

Comment:
These should really be split out into their own components. There's an awful lot going on in these functions that really would be better suited to components.